taskipy/pyproject.py

```diff
@@ -10,29 +10,47 @@
     MissingPyProjectFileError,
     MissingTaskipyTasksSectionError
 )
+from taskipy.variable import Variable
 
 
 class PyProject:
     def __init__(self, base_dir: Path):
         pyproject_path = self.__find_pyproject_path(base_dir)
-        self.__items = PyProject.__load_toml_file(pyproject_path)
+        self.__items = self.__load_toml_file(pyproject_path)
```

Comment:
Nice idea, but I think that it was better the way it was implemented before. I'll explain: `__load_toml_file` is a **static** method (belongs to the class), and therefore, IMO using `PyProject` is preferable because it indicates better that this method does not belong to the instance.

Reply:
The idea was to conform with the other change that was made, where the static method __find_pyproject_path was changed to use `self`. I can change them both to use the class name if that is the preference.

Reply:
I would love that. Thanks :)